### PR TITLE
[feature_selection] update function yaml

### PR DESCRIPTION
**Pull request by aviaIguazio**
updating file for jira fix https://iguazio.atlassian.net/browse/ML-5650

### CODE REVIEW

File: feature_selection/test_feature_selection.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-from mlrun import code_to_function
+from mlrun import code_to_function, get_dataitem
```
Comment (Eyal-Danieli) [inline feature_selection/test_feature_selection.py:15]: I don't see you use that `get_dataitem` 
```suggestion

```

File: feature_selection/item.yaml [hunk 2]
```diff
@@ -12,14 +12,14 @@ labels:
...
+platformVersion: 3.6.0
 spec:
   filename: feature_selection.py
   handler: feature_selection
   image: mlrun/mlrun
   kind: job
   requirements: []
 url: ''
-version: 1.4.0
+version: 1.7.0
```
Comment (Eyal-Danieli) [inline feature_selection/item.yaml:25]: just for curiosity why the version jumps from 1.4.0 to 1.7.0? why not 1.5.0?

**Review by Eyal-Danieli — COMMENTED** [review]
nice, a single fix

File: feature_selection/item.yaml (hunk 2 above)
Comment (aviaIguazio) [reply feature_selection/item.yaml]: you are right updated to 1.5.0

**Pull request merged**